draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1624,67 +1739,7 @@ def map2curve25519(r:felem_t) -> felem_t:
     return x
 ~~~
 
-## hash2base {#hashtobase}
-
-The following procedure implements hash2base.
-
-~~~
-hash2base(x)
-
-Parameters:
-
-  H - cryptographic hash function to use
-  hbits - number of bits output by H
-  p - order of the base field Fp
-  label - context label for domain separation
-
-Preconditions:
-
-  floor(log2(p)) + 1 >= hbits
-
-Input:
-
-  x - an octet string to be hashed
-
-Output:
-
-  y - a value in the field Fp
-
-Steps:
-
-  1. t1 = H("h2c" || label || I2OSP(len(x), 4) || x)
-  2. t2 = OS2IP(t1)
-  3. y = t2 mod p
-  4. Output y
-~~~
-
-where I2OSP, OS2IP {{RFC8017}} are used to convert an octet string to and from
-a non-negative integer, and a || b denotes concatenation of a and b.
-
-### Considerations
-
-Performance: hash2base requires hashing the entire input x. In some
```

Comment:
Why drop this section? I think we ought to keep it.

Reply:
The performance paragraph was included again now in Section 4.
The remainder of the paragraphs are also in Section 4.